cdap-ui/app/features/admin/templates/namespace/create.html

```diff
@@ -15,7 +15,7 @@
 -->
 
 <div class="modal-header clearfix">
-  <h3 class="modal-title pull-left"> Add Namespace </h3>
+  <h3 class="modal-title pull-left"> Add a Namespace </h3>
```

Comment:
It's worse than that. The buttons say "Add X". But the next screen usually says "Add a/an X", while the button to accept the add says "Create"...so I was attempting to make this one screen in alignment with the others. I will open an issue about having the rest of the interface made more consistent...